### PR TITLE
Clean up TPRs

### PR DESCRIPTION
**Pull request by andraxylia**

### CODE REVIEW

File: _docs/tasks/installing-istio.md [hunk 1]
```diff
@@ -199,6 +199,11 @@ kubectl create -f <(istioctl kube-inject -f <your-app-spec>.yaml)
    kubectl delete -f istio-rbac-alpha.yaml
    ```
 
+3. Delete Istio Kubernetes [TPRs](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-third-party-resource):
+       ```bash
+       kubectl delete -f istioconfigs --all-namespaces
```
Comment (kyessenov) [inline _docs/tasks/installing-istio.md]: I don't think this really works. Have you tried it?

**Comment by andraxylia** [comment]
I updated it, with -all.

Get works with -all-namespaces, but delete only accepts -all.

**Comment by rshriram** [comment]
Does this delete only istio config or other things as well ?

File: _docs/tasks/installing-istio.md [hunk 2]
```diff
@@ -199,6 +199,11 @@ kubectl create -f <(istioctl kube-inject -f <your-app-spec>.yaml)
    kubectl delete -f istio-rbac-alpha.yaml
    ```
 
+3. Delete Istio Kubernetes [TPRs](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-third-party-resource):
+       ```bash
+       kubectl delete -f istioconfigs --all
```
Comment (kyessenov) [inline _docs/tasks/installing-istio.md]: "-f" is incorrect

**Comment by kyessenov** [comment]
it deletes istioconfigs without removing istioconfig declaration.
`kubectl delete thirdpartyresource istioconfig` will delete the declaration.

**Pull request merged**